test/diff-details/expected-stdout

```diff
@@ -5,7 +5,6 @@ CH changed
 Diff:
 CH B bar
   "description":
-    -%c
```

Comment:
Prazdne hodnoty nevypisujeme:
- `description` je prazdny string v `remote`, vypisovalo sa to ako `-(space)`, co vyzeralo skor, ako keby tam bol prazdny riadok.